.storybook/build-story.js

```diff
@@ -65,9 +65,6 @@ function buildStory ({package: pkg, dependencies, component: Component, knobs, s
 		return (
 			<div className="story-container">
 				{dependencies && <BuildService dependencies={dependencies} />}
-				{pkg.style && <Helmet>
```

Comment:
Is the intention here that Storybook will transpile any Sass source files too?

Reply:
yeah, it kind of has to if we're importing component source files. i've set up `css-loader` and `style-loader` in storybook's webpack config and imo they work far better than including compiled styles (`x-styling-demo` loads styles instantly rather than in 2 seconds)